packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -1260,7 +1260,15 @@ class _DirectionallyExtendCaretSelectionAction<T extends DirectionalCaretMovemen
   }
 }
 
-class _SelectableRegionContainerDelegate extends MultiSelectableSelectionContainerDelegate {
+/// A delegate for the [SelectionContainer]s of [SelectableRegion]s and their
```

Comment:
This is no longer true. The doc should mention this delegate is used when selectables in the subtree are fixed in screen position and don't scroll.